Documentation/network/kubernetes/local-redirect-policy.rst

```diff
@@ -480,11 +480,17 @@ security credentials for pods.
       05:16:05.229597  In de:e4:e9:94:b5:9f ethertype IPv4 (0x0800), length 76: 192.168.98.118.47934 > 192.168.60.99.8181: Flags [S], seq 669026791, win 62727, options [mss 8961,sackOK,TS val 2539579886 ecr 0,nop,wscale 7], length 0
       05:16:05.229657 Out 56:8f:62:18:6f:85 ethertype IPv4 (0x0800), length 76: 192.168.60.99.8181 > 192.168.98.118.47934: Flags [S.], seq 2355192249, ack 669026792, win 62643, options [mss 8961,sackOK,TS val 4263010641 ecr 2539579886,nop,wscale 7], length 0
 
-Miscellaneous
-=============
-When a Local Redirect Policy is applied, cilium BPF datapath translates frontend
-(identified by ip/port/protocol tuple) address from the policy to a node-local backend
-pod selected by the policy. However, when traffic originates from the node-local
-backend pod(s), and is destined to the policy frontend, we skip translating the
-frontend address using ``sk_lookup_`` BPF helpers. This is done in order to avoid
-forming a loop. As a result, traffic in such cases is forwarded to the original frontend.
+Advanced configurations
+=======================
+When a local redirect policy is applied, cilium BPF datapath redirects traffic going to the policy frontend
+(identified by ip/port/protocol tuple) address to a node-local backend pod selected by the policy.
+However, for traffic originating from a node-local backend pod destined to the policy frontend, users may want to
+skip redirecting the traffic back to the node-local backend pod, and instead forward the traffic to the original frontend.
+This behavior can be enabled by setting the ``skipRedirectFromBackend`` flag to ``true`` in the local redirect policy spec.
+The configuration is only supported with socket-based load-balancing, and requires ``SO_NETNS_COOKIE`` feature
+available in Linux kernel version >= 5.8.
+
+.. note::
+
+    In order to enable this configuration starting Cilium version 1.16.0, previously applied local redirect policies
+    and policies selected backend pods need to be re-created.
```

Comment:
Just to clarify, when you say "re-created" here, do you mean that the policy must be explicitly deleted and then created again? This wording used here could cause folks to believe that they could just `kubectl apply` their policies again and call it a day.